spyder/plugins/shortcuts/plugin.py

```diff
@@ -63,11 +63,13 @@ class Shortcuts(SpyderPluginV2):
     def get_name():
         return _("Keyboard shortcuts")
 
-    def get_description(self):
+    @staticmethod
+    def get_description():
         return _("Manage application, widget and actions shortcuts.")
```

Comment:
```suggestion
        return _("Manage application, pane and actions shortcuts.")
```

"Widgets" isn't a term typically used in the user-facing UI